tools/rna_tools/pipmir/tool_data_table_conf.xml.sample

```diff
@@ -0,0 +1,7 @@
+<!-- Use the file tool_data_table_conf.xml.oldlocstyle if you don't want to update your loc files as changed in revision 4550:535d276c92bc-->
```

Comment:
<columns>dbkey, value</columns> vs <columns>value, dbkey, name, path</columns>

which is one is correct? the latter one is in the current version of pipmir. the former one is in extract_genomic_dna.